src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -298,6 +298,24 @@ private bool DerivesFrom(string typea, string typeb, ProtoCore.Core core)
             }
         }
 
+        /// <summary>
+        /// For checking the ML Context menu item
+        /// </summary>
+        public bool IsML
```

Comment:
Can we use something like `IsDisplayingMLRecommendation`?

Reply:
Done.